NBA-Stats-Dashboard/src/components/GeoMaps.jsx

```diff
@@ -1,10 +1,73 @@
+import USMap from './USMap';
+import WorldMap from './WorldMap';
+import { useState } from 'react';
+import '../styles/GeoMaps.css';
+
 function GeoMaps() {
-    return (
-      <div>
-        <h1>GeoMaps</h1>
-      </div>
-    );
+  const [mapSelected, setMapSelected] = useState({ us: true, world: false });
+  function mapSelection() {
+    setMapSelected((prevMapSelection) => {
+      return {
+        ...prevMapSelection,
+        us: !mapSelected.us,
+        world: !mapSelected.world,
+      };
+    });
   }
-  
-  export default GeoMaps;
-  
\ No newline at end of file
+  return (
+    <div className='geoMapsSectionContainer'>
+      <div className='mapSection mapSection1'>
+        <h1>
+          Geo-Demographics - Where are the NBA Players from? (Birthplaces)
+        </h1>
+      </div>
+      <div className='mapSection mapSection2'>
+        <p>
+          This is data on where NBA players were born. This data is
+          comprehensive of NBA players past and present up to the 2023 NBA
+          season.{' '}
+        </p>
+        <p>
+          Reference: The raw data was collected from{' '}
+          <a href='https://www.basketball-reference.com/friv/birthplaces.fcgi'>
+            basketball-reference
+          </a>
+        </p>
+      </div>
+      <div className='mapSection mapSection3'>
+        <fieldset className='mapSelectionField'>
+          <legend>MAP Selection</legend>
+          <label>
+            <input
+              id='us-map'
+              type='radio'
+              name='map'
+              value='us'
+              checked={mapSelected.us}
+              onChange={mapSelection}
+            />{' '}
+            US Map
+          </label>
+          <label>
```

Comment:
Same as above

Reply:
Fixed! Like the other situation, I used the htmlFor attribute and made sure it matches the id of the corresponding radio button input.